plugin/package.json

```diff
@@ -47,12 +48,13 @@
     "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-react": "^7.32.2",
-    "typescript": "^4.9.5"
+    "typescript": "^4.9.5",
+    "vite": "^4.4.9"
   },
   "scripts": {
-    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
-    "build": "DISABLE_ESLINT_PLUGIN=true react-scripts build",
-    "serve": "pnpx serve -s build",
+    "start": "vite",
```

Comment:
To avoid having to change configs in other places, add: `--port 3000` here.

Reply:
yup added for dev server in the vite config. 
ATM both will run at 3000